services/py-api/src/utils.py

```diff
@@ -43,3 +45,40 @@ async def stop(self) -> None:
             raise RuntimeError("The AsyncClient has not been initialized!")
 
         await self._async_client.aclose()
+
+
+class JwtUtility:
+    """A class providing methods for encoding data in with a predefined format into a JWT token, or decoding a JWT
+    token into a predefined format (TypedDict)"""
+
+    @staticmethod
+    def encode_data(data: JwtUserData) -> str:
+        return str(encode(data, key="secret", algorithm="HS256"))
```

Comment:
You're not using the actual secret as a key

Reply:
> Also, a sanity check, why do you have to cast to a string?

mypy